File: PROGRAMMERS/lv0/가위 바위 보.js
let map = {
  2: 0,
  0: 5,
  5: 2,
};

function solution(rsp) {
  return [...rsp].map((v) => map[v]).join("");
}

let inps = ["2", "205"];

for (let i of inps) {
  console.log(solution(i));
}

File: PROGRAMMERS/lv0/python/가위 바위 보.py
def solution(rsp):
    dic = {
        '2':'0',
        '0':'5',
        '5':'2'
    }
    return ''.join([dic[v] for v in rsp])

inps = [
    '2',
    '205'
]

for inp in inps:
    print(solution(inp))